openingd/dualopend.c

```diff
@@ -2382,9 +2386,13 @@ static void accepter_start(struct state *state, const u8 *oc2_msg)
 	if (!tx_state->psbt)
 		tx_state->psbt = create_psbt(tx_state, 0, 0,
 					     tx_state->tx_locktime);
-	else
+	else {
 		/* Locktimes must match! */
-		tx_state->psbt->tx->locktime = tx_state->tx_locktime;
+		tx_state->psbt->fallback_locktime = tx_state->tx_locktime;
+		if (!psbt_set_version(tx_state->psbt, 2)) {
+			status_broken("Could not set PSBT version");
```

Comment:
Needs to be status_failed I think, otherwise we stumble along and it gets worse (and never seen, since nobody reads logs!).  And should probably print the psbt?

Reply:
`negotiation_failed` perhaps?

Reply:
changed to status_failed, added printing